services/has-many-dissociator.js

```diff
@@ -1,22 +1,46 @@
 'use strict';
 var _ = require('lodash');
+var Operators = require('../utils/operators');
 var ErrorHTTP422 = require('./errors').ErrorHTTP422;
 
-function HasManyAssociator(model, association, opts, params, data) {
+function HasManyDissociator(model, association, options, params, data) {
```

Comment:
You should check if `association` exists and return a 404 if does not. Here or in forest-expres